src/rules/no-quoted-strings-inside-quote-function/README.md

```diff
@@ -0,0 +1,46 @@
+# no-quoted-strings-inside-quote-function
+
+Disallow quoted strings inside the quote function.
```

Comment:
Should this link to the Sass docs for the quote function? People might not be familiar with it.

Reply:
https://sass-lang.com/documentation/functions/string#quote